canonical/ubuntu.com
# Add filters sidenav

## Done

- Added filters sidenav for small screens
- Drive by: fixed filter icon alignment

## QA

- View the site locally in your web browser at: https://ubuntu-com-13458.demos.haus/security/cves?status=
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check that sidenav replaces original filters on small screens and that content is the same as per [design](https://www.figma.com/file/nUJEBpOPTW74RqDRuMQXxB/23.04-U.com---CVEs?type=design&node-id=949-22651&mode=design&t=iMJYHgpN4MAcNAL3-0). Note: **actual filtering functionality covered in separate ticket**

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8198
